src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Resolvers/OpenApiConfigurationResolver.cs

```diff
@@ -1,36 +1,46 @@
-using System;
-using System.Linq;
-using System.Reflection;
-
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Configurations;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
 using Microsoft.OpenApi.Models;
+using System;
+using System.Linq;
+using System.Reflection;
 
 namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Resolvers
 {
     /// <summary>
     /// This represents the resolver entity for <see cref="OpenApiServer"/>.
     /// </summary>
     public static class OpenApiConfigurationResolver
-    {
+    {
+        /// <summary>
+        /// Property used to allow for dynamic generation of OpenApiConfigurationOptions
+        /// </summary>
+        public static IOpenApiConfigurationOptions ConfigurationOptions { get; set; }
+

```

Comment:
I don't think this is necessary if the in-proc worker's `OpenApiHttpTriggerContext` class changes its constructor parameter.